models/ironbank/ethereum/ironbank_ethereum_sources.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+
+sources:
+  - name: ironbank_ethereum
+    description: "decoded events and function calls for ironbank on ethereum"
+    tables:
```

Comment:
i notice you've added a bunch of source decoded tables, however, the models added don't use any of the sources. will they be used later in another PR?

Reply:
Yes, some will be used in another PR.